README.md

```diff
@@ -29,7 +30,16 @@ Instantiate an instructor named 'Robin' and call its `givesHomework` method with
 Your Answer:
 
 ```js
-// your code here
+function Instructor(name, assignment) {
+  this.name = name;
+  this.assignment = assignment;
+  this.givesHomework = function() {
+    console.log(this.name + " gives student " +this.assignment+ " for Friday's homework.")
+  }
+}
+
+var Robin = new Instructor("Robin", "Intro to Ruby");
+Robin.givesHomework();
```

Comment:
Ah, I'm sorry. I didn't see that you were passing the assignment into the constructor function. Not quite what the instructions asked for, but I'll give it to you! :+1: